README.md

```diff
@@ -22,6 +22,8 @@ The client should be as simple as is practical to implement the desired user exp
 
 Java code should conform to the Google Java [styleguide](https://google.github.io/styleguide/javaguide.html). The project makes use of a linter and autoformatter for Java to help with this.
 
+We have an autoformatter if there isn't one in your IDE - just run `bin/fmt` and your code should be automatically formatted.
```

Comment:
Might be good to mention we only have this for Java code (at least for now...).

I'd like to add something for our HTML and TS as well if possible. I'll look into it.

Reply:
Done!